js/modal.js

```diff
@@ -40,32 +66,56 @@ const addModalComment = (miniatureData) => {
     commentElementMessage.textContent = message;
     commentsBlock.appendChild(commentElement);
 
-    const quantityOfComments = commentsBlock.children.length;
-    showedCommentsCount.textContent = quantityOfComments;
+    for (let i = 0; i < commentsBlock.children.length; i++) {
+      if (i > 4) {
+        commentsBlock.children[i].classList.add('hidden');
+      }
+    }
   });
 };
 
 const onPicturesContainerClick = (evt) => {
   const miniatureData = photosData.find(
     (photoData) => photoData.id === +evt.target.dataset.id
   );
-
-  if (evt.target.nodeName === 'IMG') {
+  if (evt.target.classList.contains('picture__img')) {
     modal.classList.remove('hidden');
-    commentsCountContainer.classList.add('hidden');
     body.classList.add('.modal-open');
     addModalContent(miniatureData);
     addModalComment(miniatureData);
+    document.addEventListener('keydown', onDocumentKeydown);
+  }
+
+  editShownCommentsCount();
+  checkIfCountersEquals();
+};
+
+const onCommentsLoaderButtonClick = () => {
+  for (
+    let i = COMMENTS_QUANTITY_CHECKER;
+    i < commentsBlock.children.length;
+    i++
+  ) {
+    if (i < COMMENTS_QUANTITY_CHECKER + NEXT_QUANTITY_OF_COMMENTS) {
+      commentsBlock.children[i].classList.remove('hidden');
+    }
```

Comment:
https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%B0%D1%8F_%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B0#:~:text=%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%B0%D1%8F%20%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B0%20(%D0%B0%D0%BD%D0%B3%D0%BB.,%D0%BF%D0%BE%20%D0%BF%D0%BE%D0%B4%D1%85%D0%BE%D0%B4%D1%8F%D1%89%D0%B8%D0%BC%20%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B0%D0%BC%20%D0%B8%20%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D1%8F%D0%BC.